youtube_dl/extractor/rtve.py

```diff
@@ -27,7 +27,7 @@
 class RTVEALaCartaIE(InfoExtractor):
     IE_NAME = 'rtve.es:alacarta'
     IE_DESC = 'RTVE a la carta'
-    _VALID_URL = r'https?://(?:www\.)?rtve\.es/(m/)?(alacarta/videos|filmoteca)/[^/]+/[^/]+/(?P<id>\d+)'
+    _VALID_URL = r'https?://(?:www\.)?rtve\.es/(m/)?((alacarta|playz?)/videos|filmoteca)/[^/]+/[^/]+/(?P<id>\d+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?rtve\.es/(m/)?((alacarta|play?)/videos|filmoteca)/[^/]+/[^/]+/(?P<id>\d+)'
```

Reply:
Note that `playz` redirects to `play`. If I remove the `z`, it doesn't work with `playz`, which is the link shared by RTVE in a lot of places. Also, note that your change doesn't makes sense because you are specifiying here that the character `y` is optional, which is not true.